### PR TITLE
Microsoft CAS fix example in yml file

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/35493

## Description
Fixed the erroneous example in the yml file regarding the syntax of the filters.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Noy-Maimon** [comment]
@roysagi Can you please take a look the the .yml file and the RN? Thanks

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml [hunk 1]
```diff
@@ -61,8 +61,9 @@ configuration:
   type: 16
 - additionalinfo: 'A custom filter by which to filter the returned files. If you pass
     the custom_filter argument it will override the other filters from the integration
-    instance configuration. Example for Custom Filter: {"entity.policy":{"eq":"Impossible
-    travel"}}. For more information about filter syntax, please refer to https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters.'
+    instance configuration. Example for Custom Filter: {"severity":{"eq":2}}.
+    Note that for filtering by "entity.policy", one should use the id of the policy.
```
Comment (roysagi) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml:65]: Is there a way to give an example for the use of `entity.policy` this seems a bit vague

Comment (Noy-Maimon) [reply Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml:65]: example was added

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml [hunk 2]
```diff
@@ -61,8 +61,14 @@ configuration:
...
+    For example, for retrieving the policy:
+    {"policyType": "ANOMALY_DETECTION", "id": "5e6fa96cb5172297ca756554", "label": "Impossible travel",
```
Comment (roysagi) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml:67]:
```suggestion
    {"policyType": "ANOMALY_DETECTION", "id": "1234", "label": "Impossible travel",
```

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml [hunk 3]
```diff
@@ -61,8 +61,14 @@ configuration:
...
+     "type": "policyRule"}"
+    please query on {"entity.policy":{"eq":5e6fa96cb5172297ca756554}}.
```
Comment (roysagi) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml:69]:
```suggestion
    please query on {"entity.policy":{"eq":1234}}.
```

**Review by roysagi — COMMENTED** [review]
I suggest shortening the id as it appears in additional info - also might be perceived as a secret

**Review by reutshal — APPROVED** [review]

**Pull request merged**